src/animation.cpp

```diff
@@ -20,7 +20,7 @@ point relative_view_pos(player const &u, point const &p) noexcept
 
 void draw_animation_delay(long const scale = 1)
 {
-    auto const delay = static_cast<long>(OPTIONS["ANIMATION_DELAY"]) * scale * 1000000l;
```

Comment:
What exactly does clang think is ambiguous here? The value from the options is a long (after the cast), `scale` is already, and `1000000l` is, too. There are only `long`s involved here.